### DISCUSSION TITLE
Starting Brainvisa from Virtual Toolbox failed

### DISCUSSION THREAD

**Discussion by francoisLeroy** [brainvisa/brainvisa.github.io#175] [Q&A]
After installing the Brainvisa Virtual Toolbox kit, I cannot start Brainvisa. It get stuck repeating the same message shown in the enclosed png picture. Here is also the Vbox log file.
Thanks for your help
![BV_VM_error](https://github.com/brainvisa/brainvisa.github.io/assets/16325253/1dc0a476-0715-4632-a361-fa2e9bb9f149)
[VBox.log](https://github.com/user-attachments/files/15954698/VBox.log)

**Comment by denisri** [comment]
Hi,
You are talking about the VirtualBox version of BrainVisa 5.1.1, right ? On a Windows host system I suppose ? Your description is not very precise so I have to guess...
Actually I don't understand where / when the problem happens: VirtualBox should start a Linux VM, where you can open a linux shell or something, and run brainvisa inside it, but here in your error message snapshot I don't recognize anything, it rather looks like Windows error messages (are they ?) So it it at the VM startup, or does the machine start, and the problem happen when starting brainvisa ? In the latter case, does another 3D program, like glxgears, run in the VM ?
To precise my question above, is it the whole VM (VirtualBox) which crashes, or brainvisa inside it ?

**Comment by francoisLeroy** [comment]
Thanks Denis for your help
Yes, it is Brainvisa 5.1.1. on my Windows 10 laptop.
I can start the Oracle Virtual VM software (release 6.1). When I press to start Brainvisa, I see the Ubuntu downloading within the VM window. The message within the png snapshot file that I sent is written right after the Ubuntu downloading and before any possible end user interactions. I don't think any other 3D program is running in the VM apart from Brainvisa.
Should I try to reinstall/upgrade the Oracle VirtualBox, or/and the Brainvisa toolbox?
Thanks again
Francois

**Comment by denisri** [comment]
Hi François,
Sorry but it's still not clear... I just don't understand what you are saying by "downloading ubuntu after starting the virtual machine".
If I understand a bit, you have downloaded the .ova image, then you run VirtualBox, and import the .ova image. Thus there is no download any longer from now: if you can start the machine, it's already downloaded.
- Does the error occur during image **importation** into VirtualBox ? If so, maybe check that the image has been correctly downloaded without corruption.
- Or if the image import ends correctly, you see the VM in VirtualBox list of machines to be started
- Then you start the VM in VirtualBox. Does it crash when **booting** this virtual machine ?
- or does the VM boot correctly, so you have a working Linux (Ubuntu), and then does the problem happen when actually **starting the `brainvisa` software** ?

I'm sorry but it is very difficult to understand what you have done and when the problem occurs, you have to be very precise in the description...

**Comment by francoisLeroy** [comment]
I think the BV ova image import is successful. 
Here is the Brainvisa VM status
![bv_vm](https://github.com/brainvisa/brainvisa.github.io/assets/16325253/120c8194-c072-4e15-8800-bbdbdc83f4c0)
The "Ubuntu" name appears after booting the Brainvisa VM then a few other things and then error occurs. 
I hope this is clearer.

**Comment by denisri** [comment]
So it's when booting the virtual machine.
Maybe you can check and adjust some settings in the VM config, like the display options. Here we see that the emulated display has 50 Mb of memory, which seems very small, you'd probably better increase that (at least). But maybe also a few other options deserve to be checked.

**Comment by francoisLeroy** [comment]
I increased the emulated display to 128Mb but the error occurs. 
In fact, it is always the same error message that is repeated two or three times before the display being stuck on the first snapshot image i sent
What would be the other possible options to change?
Best

**Comment by sapetnioc** [comment]
I suppose you are using a laptop managed by CEA. If yes, you have no other choice than VirtualBox because you cannot install WSL2. If not, using WSL2 is much more comfortable and simple than VirtualBox (to my opinion).

For VirtualBox, I do not precisely remember the proposed options but graphical configuration is important (memory, type of graphic controller, activation of 3D acceleration, etc.). We may find a moment to see that together (face to face or remotely).

**Comment by francoisLeroy** [comment]
I uninstalled Oracle VirtualBox and reinstalled it. I imported the Brainvisa ova file again and everything seems to work fine now.
I guess the error was caused by a change in login name on the laptop (because of CEA Gaia installation a few weeks ago)
Thanks a lot for your support
See you soon on the 5th of July maybe?
F

**Comment by denisri** [comment]
So it's fixed. Great :)
